apps/studio/components/interfaces/DataWarehouse/WarehouseCollectionDetail.tsx

```diff
@@ -14,6 +13,36 @@ import { useFlag } from 'hooks/ui/useFlag'
 import { Button } from 'ui'
 import LogTable from '../Settings/Logs/LogTable'
 import { TestCollectionDialog } from './TestCollectionDialog'
+import { Tooltip, TooltipContent, TooltipTrigger } from '@ui/components/shadcn/ui/tooltip'
+import { Input } from '@ui/components/shadcn/ui/input'
+import DatePickers from '../Settings/Logs/Logs.DatePickers'
+import { DatetimeHelper } from '../Settings/Logs/Logs.types'
+import dayjs from 'dayjs'
+
+const dayjsBase = dayjs()
+const INTERVALS: DatetimeHelper[] = [
+  {
+    text: 'Last 1 hour',
```

Comment:
keep it consistent with logs explorer
![image](https://github.com/user-attachments/assets/1dc33c7d-02cd-477e-bf97-6e94b8f1a150)

```suggestion
    text: 'Last hour',
```